### ISSUE TITLE
Bug: Not sending IR TV commands from plugin

### ISSUE THREAD

**Issue by ddanetidwell** [OpenWonderLabs/homebridge-switchbot#553]
### Describe The Bug

Plugin is not sending IR commands to custom TV configuration

### To Reproduce

Press button for TV power on/off

### Expected behavior

TV turns on/off

### Relevant log output

```shell
10/21/2022, 2:20:50 PMHomeConnectSwitchBotWARNINGTV: TV Connection Type: undefined, commands will not be sent to OpenAPI
10/21/2022, 2:20:55 PMHomeConnectSwitchBotWARNINGTV: TV Connection Type: undefined, commands will not be sent to OpenAPI
10/21/2022, 2:21:20 PMHomeConnectSwitchBotWARNINGTV: TV Connection Type: undefined, commands will not be sent to OpenAPI
10/21/2022, 2:43:29 PMHomeConnectSwitchBotWARNINGTV: TV Connection Type: undefined, commands will not be sent to OpenAPI
10/21/2022, 2:43:34 PMHomeConnectSwitchBotWARNINGTV: TV Connection Type: undefined, commands will not be sent to OpenAPI
```

### Config for homebridge-switchbot

```shell
NA
```

### Device & Model

NA

### Node.js Version

16.18.0

### NPM Version

Unk

### Homebridge Version

1.5.0

### Homebridge Switchbot Plugin Version

2.2.1

### Operating System

Raspbian

**Comment by donavanbecker** [comment]
If you don't post config, I can't help

**Comment by donavanbecker** [comment]
Fixed in v2.2.2
